### PR TITLE
Added the option to customize the label of the submit button

### PR DESCRIPTION
**Pull request by mkotsollaris**

### CODE REVIEW

File: src/components/TruliooForm.jsx [hunk 1]
```diff
@@ -54,7 +54,9 @@ export class TruliooForm extends React.Component {
           onChange={(e) => this.handleChange(e)}
           onSubmit={(e) => this.handleSubmit(e)}
           formData={formData}
-        />
+        >
+          <button className="btn btn-info" type="submit">{this.props.buttonName}</button>
```
Comment (kateika) [inline src/components/TruliooForm.jsx]: type submit is default type for button, so you can delete it if you want

File: src/tests/unit/EmbedID.test.jsx [hunk 2]
```diff
@@ -130,4 +130,26 @@ describe('EmbedID events function properly', () => {
     const svg = container.querySelector('svg');
     expect(svg.getAttribute('data-prefix')).toBe('fas');
   });
+
+  it('renders EmbedID with pre-defined button label', async () => {
+    const proxyURL = 'http://localhost:3111';
```
Comment (kateika) [inline src/tests/unit/EmbedID.test.jsx]: that's why you changed BASE_URL to http://localhost:8855 ?

File: src/tests/unit/EmbedID.test.jsx [hunk 3]
```diff
@@ -130,4 +130,26 @@ describe('EmbedID events function properly', () => {
...
+    const { container } = render(
+      <EmbedID
+        url={proxyURL}
+        getFields={jest.fn()}
+        submitForm={jest.fn()}
+        buttonName={buttonName}
+        uiSchema={uiSchema}
+      />,
+    );
+    expect(getByText(container, new RegExp(buttonName)));
```
Comment (kateika) [inline src/tests/unit/EmbedID.test.jsx]: why do you need RegExp here? You have shared button that can contain any text?

File: src/components/TruliooForm.jsx (hunk 1 above)
Comment (mkotsollaris) [reply src/components/TruliooForm.jsx]: 👍

File: src/tests/unit/EmbedID.test.jsx (hunk 3 above)
Comment (mkotsollaris) [reply src/tests/unit/EmbedID.test.jsx]: 👍

**Pull request merged**